comp/otelcol/converter/README.md

```diff
@@ -24,8 +24,32 @@ If it finds datadogexporters which are not defined in a pipeline with the promet
 
 For any prometheus receiver collecting collector health metrics, and sending these to Datadog, it will update the job name to `datadog-agent`. This ensures the health metrics are tagged by `service:datadog-agent` and differentiable from collector health metrics.
 
+### Datadog Connector
+
+The converter will automatically set `datadogconnector` config `trace.span_name_as_resource_name` to true in any datadog connectors in your configuration.
+
 ## Provided and enhanced config
 
 `GetProvidedConf` and `GetEnhancedConf` return the string representation of the user provided and autoconfigured conf respectively. Currently, these APIs have two limitations:
 - They do not redact sensitive data
 - They do not provide the effective config (including defaults...etc)
+
+## Opting out of converter
+
+It is possible to opt out of the converter by setting env var `DD_OTELCOLLECTOR_CONVERTER_ENABLED` or agent config `otelcollector.converter.enabled` to `false` (`true` by default). Please note that by doing so, you are removing functionality including flare collection from otel-agent, health metrics from collector, or infra level tagging on your telemetry data. If you want to opt out of some components, you can disable all and add the components that you require manually:
+
+### Extensions
+
+Please refer to the following example in order to manually set the `pprof`, `health_check`, `zpages` and `datadog` extensions: [extensions.yaml](examples/extensions.yaml). Please refer to the extensions README.md for additional information about the components:
+- [pprof](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/pprofextension/README.md): Enables collecting collector profiles at a defined endpoint.
+- [health_check](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/healthcheckextension/README.md): Enables an HTTP url that can be probed to check the status of the OpenTelemetry Collector.
+- [zpages](https://github.com/open-telemetry/opentelemetry-collector/blob/main/extension/zpagesextension/README.md): Enables an extension that serves zPages, an HTTP endpoint that provides live data for debugging different components
+- [datadog](../extension/README.md): Enables otel-agent information to be collected in the datadog-agent flare.
+
+### Prometheus Receiver
```

Comment:
how come this receiver and the infra processor have two sections each?

Reply:
Are you referring to the fact that prometheus receiver and infraattributes processor are both under `## Autoconfigure logic` and `## Opting out of converter` ? If so, the first explains in which manner the converter will add the section, whereas the latter is the addition of instructions on how to add the component manually.